cl_dll/hud_countdown.h

```diff
@@ -10,6 +10,7 @@ class CHudCountdown : public CHudBase
 	char name2[32];
 
 	cvar_t* cl_autorecord;
+	cvar_t* cl_autorecord_stop;
```

Comment:
1. Since it's not related to countdown, but rather `CHud` itself, it's better to put it into `CHud` (look for `m_pCvarColor` in `hud.h` and `hud.cpp` and put around there).

2. When you do this you'll be able to access the `cl_autorecord_stop` variable directly, so change `CVAR_GET_FLOAT()` in `CHud::Redraw()` to direct access as it's faster and less error-prone (`cl_autorecord_stop->value > 0.0f`).

3. `cl_autorecord_stop` doesn't really make sense as a name, it's a general demo auto-stop behavior which can be used apart from auto-record. Maybe `cl_autostop`?